mppm/virtual-players/virtual-players-focus.md

```diff
@@ -31,4 +31,4 @@ Use the following keyboard shortcuts to switch focus to a specific virtual playe
 | **Ctrl**+**F9** (macOS: **Cmd**+**F9**)   |  Focus on **Player 1**.   |
```

Comment:
Are the keyboard commands accurate? I couldn't get them to work.

Reply:
They should be accurate for this release, yes. These are new additions and I took these definitions straight from the code so they definitely should work!